hbase-server/src/main/java/org/apache/hadoop/hbase/quotas/MasterQuotasObserver.java

```diff
@@ -65,8 +81,11 @@ public void postDeleteTable(
     }
     final Connection conn = ctx.getEnvironment().getConnection();
     Quotas quotas = QuotaUtil.getTableQuota(conn, tableName);
+    Quotas quotasAtNamespace = QuotaUtil.getNamespaceQuota(conn, tableName.getNamespaceAsString());
```

Comment:
Maybe rename these to be `tableQuotas` and `namespaceQuotas`.

Reply:
done